hyperhtml.js

```diff
@@ -519,193 +853,34 @@ var hyperHTML = (function () {'use strict';
-    this[EXPANDO] = {s: statics, u: updates};
-    return remapping ? this : update.apply(this, arguments);
-  }
-
-  // -------------------------
-  // local variables
-  // -------------------------
-
-  var
-    ELEMENT_NODE = 1,
-    ATTRIBUTE_NODE = 2,
-    TEXT_NODE = 3,
-    COMMENT_NODE = 8,
-    DOCUMENT_FRAGMENT_NODE = 11,
-    // some attribute might be present on the element prototype but cannot be set directly
-    // TODO: following RegExp used in lukeTreeWalker switch too
-    //       if modified, remember to go there and remove its usage.
-    SHOULD_USE_ATTRIBUTE = /^style$/i,
-    // avoids WeakMap to avoid memory pressure, use CSS compatible syntax for IE
-    EXPANDO = '_hyper_html: ',
-    // use a pseudo unique id to avoid conflicts and normalize CSS style for IE
-    uid = EXPANDO + ((Math.random() * new Date) | 0) + ';',
-    // use comment nodes with pseudo unique content to setup
-    uidc = '<!--' + uid + '-->',
-    // if attributes order is shuffled, threat the browser differently
-    IE = (function (p) {
-      p.innerHTML = '<i data-i="" class=""></i>';
-      return /class/i.test(p.firstChild.attributes[0].name);
-    }(
-      // beside the initial feature detection
-      // the document could be swap-able at runtime
-      (hyperHTML.document = document).createElement('p'))
-    ),
-    appendNodes = 'append' in document ?
-      function (node, childNodes) {
-        node.append.apply(node, childNodes);
-      } :
-      function appendNodes(node, childNodes) {
-        for (var
-          i = 0,
-          length = childNodes.length;
-          i < length; i++
-        ) {
-          node.appendChild(childNodes[i]);
-        }
-      },
-    no = IE && new RegExp('([^\\S][a-z]+[a-z0-9_-]*=)([\'"])' + uidc + '\\2', 'g'),
-    comments = IE && function ($0, $1, $2) {
-      IEAttributes.push($1.slice(1, -1));
-      return $1 + $2 + uid + $2;
-    },
-    // verify empty textContent on .wire() setup
-    trim = EXPANDO.trim || function () {
-      return this.replace(/^\s+|\s+$/g, '');
-    },
-    // convert DOM.childNodes into arrays to avoid
-    // DOM mutation backfiring on loops
-    slice = [].slice,
-    // used for weak references
-    // if WeakMap is not available
-    // it uses a configurable, non enumerable,
-    // quick and dirty expando property.
-    wm = typeof WeakMap === typeof wm ?
-      {
-        get: function (obj) { return obj[EXPANDO]; },
-        set: function (obj, value) {
-          Object.defineProperty(obj, EXPANDO, {
-            configurable: true,
-            value: value
-          });
-        }
-      } :
-      new WeakMap(),
-    remapping = false,
-    IEAttributes
-  ;
-
-  // -------------------------
+  // ---------------------------------------------
   // ⚡️ ️️The End ➰
-  // -------------------------
+  // ---------------------------------------------
   return hyperHTML;
 
-}());
+}(document));
 
 // umd.KISS
 try { module.exports = hyperHTML; } catch(o_O) {}
```

Comment:
if legacy, these are also hard to update so no legacy project would adopt hyperHTML ? :smiley: